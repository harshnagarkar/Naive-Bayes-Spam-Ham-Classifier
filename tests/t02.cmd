  training_ham.txt training_spam.txt testing_ham2.txt testing_spam.txt 2 
