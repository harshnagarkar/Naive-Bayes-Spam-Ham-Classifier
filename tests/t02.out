ham -190.263 -215.561
ham -128.199 -140.594
ham -287.566 -301.982
ham -514.302 -548.09
ham -365.905 -388.763
ham -282.325 -308.609
ham -272.044 -285.476
ham -204.922 -213.098
ham -367.928 -405.935
ham -237.367 -258.625
ham -363.744 -391.202
ham -377.207 -407.912
ham -237.195 -249.118
ham -497.764 -531.616
ham -0.247836 -1.51635
spam -418.863 -413.479
spam -1259.28 -1175.65
ham -589.873 -594.022
ham -1157.65 -1167.06
ham -752.15 -770.599
ham -511.072 -520.017
spam -150.383 -148.382
spam -111.22 -110.552
ham -223.553 -226.904
ham -396.397 -403.849
spam -94.4871 -93.1891
ham -21.8099 -23.4092
ham -816.672 -830.201
ham -603.129 -617.601
ham -656.388 -669.836
ham -524.045 -534.024
ham -669.278 -683.601
ham -0.247836 -1.51635
0.6 0.535714 0.545455
